workflow/dumping-workflows.rst

```diff
@@ -0,0 +1,30 @@
+.. index::
+    single: Workflow; Dumping Workflows
+
+How to Dump Workflows
+=====================
+
+To help you debug you could dump a representation of your workflow with
```

Comment:
`To help you debug you could dump a ...` -> `To help you debug your workflows, you can dump a ...`